packages/file-format-ts/index.ts

```diff
@@ -1,7 +1,3 @@
-import * as FileFormat1 from './v1-types.js'
-import * as FileFormat2 from './v2-types.js'
-import * as FileFormat3 from './v3-types.js'
+import * as FileFormat from './types.js'
```

Comment:
This PR changes the file from `./types.js` to `./type.ts`

Reply:
👀 … I don't follow. `./v3-types.js` is now `type.js`, so the extensions haven't changed.